packages/legend-art/style/base/_panel.scss

```diff
@@ -42,7 +42,6 @@
   &__header__actions {
     @include flexVCenter;
 
-    display: flex;
```

Comment:
`display: flex` was redundant since this style already has `@include flexVCenter;` so I removed it here.